<?xml version="1.0" encoding="UTF-8"?>
<!-- Generated by org.testng.reporters.JUnitReportReporter -->
<testsuite hostname="WKWZTPqvmdckJHX" failures="0" tests="11" name="com.qa.abreva.test.AbrevaColdSoreConfidenceTest" time="180.852" errors="1" timestamp="2023-05-16T23:32:44 IST" skipped="2">
  <testcase classname="com.qa.abreva.test.AbrevaColdSoreConfidenceTest" name="cookieButton" time="8.777"/>
  <system-out/>
  <testcase classname="com.qa.abreva.test.AbrevaColdSoreConfidenceTest" name="verifyBreadcrumbOne" time="10.390"/>
  <system-out/>
  <testcase classname="com.qa.abreva.test.AbrevaColdSoreConfidenceTest" name="verifyHyperLinkHere" time="11.571"/>
  <system-out/>
  <testcase classname="com.qa.abreva.test.AbrevaColdSoreConfidenceTest" name="verifyHyperLinkReadMore" time="11.755"/>
  <system-out/>
  <testcase classname="com.qa.abreva.test.AbrevaColdSoreConfidenceTest" name="verifyWholeCardSecOne" time="15.080"/>
  <system-out/>
  <testcase classname="com.qa.abreva.test.AbrevaColdSoreConfidenceTest" name="verifyWholeCardSecTwo" time="26.222"/>
  <system-out/>
  <testcase classname="com.qa.abreva.test.AbrevaColdSoreConfidenceTest" name="verifyWholeCardSecThree" time="12.987"/>
  <system-out/>
  <testcase classname="com.qa.abreva.test.AbrevaColdSoreConfidenceTest" name="verifyGetcoupon" time="26.430">
    <skipped/>
  </testcase> <!-- verifyGetcoupon -->
  <system-out/>
  <testcase classname="com.qa.abreva.test.AbrevaColdSoreConfidenceTest" name="verifyGetcoupon" time="23.234">
    <skipped/>
  </testcase> <!-- verifyGetcoupon -->
  <system-out/>
  <testcase classname="com.qa.abreva.test.AbrevaColdSoreConfidenceTest" name="verifyGetcoupon" time="25.450">
    <error message="HyperLink Get coupon is not presented in the page" type="java.lang.Exception">
      <![CDATA[java.lang.Exception: HyperLink Get coupon is not presented in the page
at com.qa.abreva.pages.AbrevaColdSoreConfidencePage.clickGetcoupon(AbrevaColdSoreConfidencePage.java:167)
at com.qa.abreva.test.AbrevaColdSoreConfidenceTest.verifyGetcoupon(AbrevaColdSoreConfidenceTest.java:118)
at java.base/jdk.internal.reflect.DirectMethodHandleAccessor.invoke(DirectMethodHandleAccessor.java:104)
at java.base/java.lang.reflect.Method.invoke(Method.java:577)
at org.testng.internal.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:133)
at org.testng.internal.TestInvoker.invokeMethod(TestInvoker.java:598)
at org.testng.internal.TestInvoker.retryFailed(TestInvoker.java:213)
at org.testng.internal.MethodRunner.runInSequence(MethodRunner.java:58)
at org.testng.internal.TestInvoker$MethodInvocationAgent.invoke(TestInvoker.java:824)
at org.testng.internal.TestInvoker.invokeTestMethods(TestInvoker.java:146)
at org.testng.internal.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:146)
at org.testng.internal.TestMethodWorker.run(TestMethodWorker.java:128)
at java.base/java.util.ArrayList.forEach(ArrayList.java:1511)
at org.testng.TestRunner.privateRun(TestRunner.java:794)
at org.testng.TestRunner.run(TestRunner.java:596)
at org.testng.SuiteRunner.runTest(SuiteRunner.java:377)
at org.testng.SuiteRunner.runSequentially(SuiteRunner.java:371)
at org.testng.SuiteRunner.privateRun(SuiteRunner.java:332)
at org.testng.SuiteRunner.run(SuiteRunner.java:276)
at org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:53)
at org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:96)
at org.testng.TestNG.runSuitesSequentially(TestNG.java:1212)
at org.testng.TestNG.runSuitesLocally(TestNG.java:1134)
at org.testng.TestNG.runSuites(TestNG.java:1063)
at org.testng.TestNG.run(TestNG.java:1031)
at org.testng.remote.AbstractRemoteTestNG.run(AbstractRemoteTestNG.java:115)
at org.testng.remote.RemoteTestNG.initAndRun(RemoteTestNG.java:251)
at org.testng.remote.RemoteTestNG.main(RemoteTestNG.java:77)
]]>
    </error>
  </testcase> <!-- verifyGetcoupon -->
  <system-out/>
  <testcase classname="com.qa.abreva.test.AbrevaColdSoreConfidenceTest" name="verifyBuyNow" time="8.956"/>
  <system-out/>
</testsuite> <!-- com.qa.abreva.test.AbrevaColdSoreConfidenceTest -->
